### PR TITLE
Indigo release candidate

### PR DESCRIPTION
**Pull request by fmessmer**
I don't understand what has happened here!
Somehow, we do not have changelogs for versions 0.6.12 - 0.6.14
@ipa-fmw, can you recall whether there was a problem when releasing this repo?

---
__Update___
Ok, I fixed it to the best of my knowledge
It seems that somehow ```kinetic_dev``` branch has been merged (or force-pushed) into ```indigo_dev``` or the ```indigo_release_candidate``` has been pushed from ```kinetic_dev``` or something...

### CODE REVIEW

File: cob_frame_tracker/CHANGELOG.rst [hunk 1]
```diff
@@ -57,6 +66,61 @@ Changelog for package cob_frame_tracker
...
+* code styling cob_frame_tracker
+* Implemented proposals from discussion https://github.com/ipa320/cob_control/pull/38. Removed tabs. Corrected node handles.
+* update changelog
+* Removed tracking error publisher / subscriber and removed additional p gain for PD-Control (already done in FrameTracker with PID controller)
+* bug fix
+* - Added a publisher for the tracking errors to send them to cob_twist_controller
+  - Added a subscriber to collect the errors and put them to the solver.
+  - Added a parameter to set the p gain. If 0.0 old behavior is active (default value).
+* Contributors: Marco Bezzon, ipa-fxm, ipa-fxm-cm, ipa-fxm-mb
+
```
Comment (mgruhler) [inline cob_frame_tracker/CHANGELOG.rst]: All this here is wrong, there still is the duplicate 0.6.12 tag, and this one includes 0.6.9 to 0.6.11

File: cob_model_identifier/CHANGELOG.rst [hunk 2]
```diff
@@ -28,6 +37,33 @@ Changelog for package cob_model_identifier
...
+* cleanup
+* sort dependencies
+* update with ipa320
+* review dependencies
+* code styling cob_model_identifier
+* beautify and code-review
+* fixing eigen linker error
+* update changelog
+* Contributors: ipa-fxm
+
```
Comment (mgruhler) [inline cob_model_identifier/CHANGELOG.rst]: as above

**Review by mgruhler — CHANGES_REQUESTED** [review]

File: cob_frame_tracker/CHANGELOG.rst (hunk 1 above)
Comment (fmessmer) [reply cob_frame_tracker/CHANGELOG.rst]: just forgot to delete this part after moving it to the top

File: cob_model_identifier/CHANGELOG.rst (hunk 2 above)
Comment (fmessmer) [reply cob_model_identifier/CHANGELOG.rst]: just forgot to delete this part after moving it to the top

**Review by mgruhler — APPROVED** [review]
fine now

**Pull request merged**